### ISSUE TITLE
'ConanApp' object has no attribute 'pyreq_loader' with conan 1.18.2

### ISSUE THREAD

**Issue by thomthom** [conan-io/conan-package-tools#521]
### Description of Problem, Request, or Question

I was trying to build a recipe on my machine using conan `1.18.2` (matching what our build agentd currently are using). But I ran into an error:

```
Version: 0.34.1
 >> Branch detected
Traceback (most recent call last):
  File "/Users/tthomas2/src/git/thirdparty_conan/scripts/recipes/ruby-build.py", line 38, in <module>
    build()
  File "/Users/tthomas2/src/git/thirdparty_conan/scripts/recipes/ruby-build.py", line 13, in build
    builder = ConanMultiPackager()
  File "/Users/tthomas2/conan-sandbox/lib/python3.6/site-packages/cpt/packager.py", line 220, in __init__
    conanfile = load_cf_class(os.path.join(self.cwd, self.conanfile), self.conan_api)
  File "/Users/tthomas2/conan-sandbox/lib/python3.6/site-packages/cpt/packager.py", line 55, in load_cf_class
    conan_api.app.pyreq_loader.enable_remotes(remotes=remotes)
AttributeError: 'ConanApp' object has no attribute 'pyreq_loader'
```

When stepping through the code in VSCode I don't see `pyreq_loader` being set by `conan_api.create_app()`.

When I tried 1.21.3 I didn't see the error. I found that strange as the `load_cf_class` seems to conditionally handle conan versions going back quite a bit beyon the 1.18 series.

### Environment Details
* Conan Package Tools Version: **0.34.1**
* Operating System: **macOS**
* Operation System Version: **Mojave**
* Conan version: **conan 1.18.2**
* Python version: **python 3.6.6**

**Comment by Pro** [comment]
Same here with conan version 1.19.0

Is there a fix for this, except updating conan?

**Comment by memsharded** [comment]
Hi @Pro 

conan-package-tools is not stable yet, and latest version keeps in sync with the latest Conan version, in this case, 1.31.1. If you are using an older Conan version, you need to keep using an older conan-package-tools version. We are trying to make conan-package-tools more Conan-version agnostic, but that will take time.

**Comment by uilianries** [comment]
Fixed by #551

It should be available on CPT 0.35.1
